src/screens/SurveyDetail/QuestionBuilders/buildQuestionList.js

```diff
@@ -0,0 +1,21 @@
+// Build question list and answers from the api for easier of question and
```

Comment:
@rafayet-monon Please correct me if I'm wrong, as I understand that 
- `determineQuestionType` render a question according to the question type
- `buildQuestionList` function build the question list from the API response
- `filterQuestionList` function filter questions out from the API response
- `questionProperties` function prepare property for the question to make it ready to render

so IMO we can
- move `buildQuestionList` and `filterQuestionList` to helpers
- make `determineQuestionType` a subcomponent
- `questionProperties` can either be a helper function or a private function in question component

what do you think?

Reply:
@carryall Fixed your concerns in [25c0413](https://github.com/rafayet-monon/surveyor/pull/28/commits/25c0413341e4022d58b56479fe2520743df8fb71)